File: P10/P29094_en/main.py
from yogi import *

m, n = scan(int), scan(int)
while m is not None and n is not None:
  v = []
  for i in range(0, n): v.append(read(float))
  print(position_maximum(v, m))
  m, n = scan(int), scan(int)

File: P10/P29094_en/main.c
#include <stdio.h>

int position_maximum(double v[], int m);

int main() {
	int m, n;
	while (scanf("%d %d", &m, &n) > 0) {
    double v[n];
    for (int i = 0; i < n; ++i) scanf("%lf", &v[i]);
    printf("%d\n", position_maximum(v, m));
	}
	return 0;
}
